addons/website_payment/controllers/main.py

```diff
@@ -78,8 +78,8 @@ def confirm(self, **kw):
         tx_id = request.session.pop('website_payment_tx_id', False)
         if tx_id:
             tx = request.env['payment.transaction'].browse(tx_id)
-            status = (tx.state == 'done' and 'success') or 'danger'
-            message = (tx.state == 'done' and 'Your payment was successful! It may take some time to be validated on our end.') or 'OOps! There was a problem with your payment.'
+            status = (tx.state == _('done') and _('success')) or _('danger')
```

Comment:
tx.state is a selection field, so these are selection values and should not be translated.